v1.1.1/js/Sidebar.js

```diff
@@ -234,6 +256,28 @@ function Sidebar(loopy){
 		}
 	});
 
+    subscribe("node/changeColorRainbow", function() {
+        self.changePalette(0);
+    });
+
+    subscribe("node/changeColorTol", function() {
+        self.changePalette(1);
+    });
+	
+    subscribe("node/changeColorWong", function() {
+        self.changePalette(2);
+    });
+
+	subscribe("node/changeColorBW", function() {
+        self.changePalette(3);
+    });
+
+	self.changePalette = function(n){
+        Node.defaultPalette = n;
+        this.pages[0].components[2].dom.innerHTML = "<div class=\"component_label\">Color:</div><div class=\"component_slider\"><img draggable=\"false\" src=\"css/sliders/"+Sidebar.COLORSLIDEROPTIONS[Node.defaultPalette]+".png\" class=\"component_slider_graphic\"><img draggable=\"false\" src=\"css/sliders/slider_pointer.png\" class=\"component_slider_pointer\"></div>"
+		// the line above SUCKS, but it changes the slider image which is what we need
```

Comment:
I can only imagine that this is the line causing it, but I've noticed something concerning that definitely needs to be fixed before we merge anything; Sometimes, you just can't change the color of a node after you change palettes. I've been able to replicate the issue by doing the following steps:
1) Load up a fresh loopy
2) Before doing anything else, click a node and change its color (this one works fine)
3) Click somewhere on the blank part of the canvas to go back to the default edit sidebar
4) Pick a different color palette
5) Click on any node that already exists
6) Try to change the color of that node, and observe that it won't let you

I'm wondering if this might have something to do with the fact that the position of the slider arrow gets reset when we change color palettes, and therefore the arrow index and the color the node is actually showing don't match up, or if it's just an unrelated thing that we should also fix.

Also, more of a nitpick than anything else, but we could definitely break up the long line into several lines for readability, and I think that might be a good idea, assuming that this line stays all long like this after the above issue is fixed.